d365fo.tools/functions/import-d365aaduser.ps1

```diff
@@ -167,6 +167,12 @@ function Import-D365AadUser {
         [string] $AadGroupId
     )
 
+    $azureADModule = Get-Module -Name "AzureAD" -ListAvailable
+    if ($null -eq $azureADModule) {
+        Write-PSFMessage -Level Host -Message "AzureAD module is not installed. Trying to install it now." -Color Yellow
+        Install-Module -Name "AzureAD" -Force -AllowClobber -Scope CurrentUser
+    }
+
```

Comment:
:question:

@Splaxi What do you think about this "just in time" installation of dependencies? I did not have a chance to fully test it yet.

Reply:
Let's have a offline chat at some point in time, over the summer - to see where we end up.

For now I'm against JIT installation...